src/server/handlers/consents.ts

```diff
@@ -0,0 +1,90 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation The Mojaloop files are made available by the Mojaloop Foundation
+ under the Apache License, Version 2.0 (the 'License') and you may not
+ use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in
+ writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS
+ OF ANY KIND, either express or implied. See the License for the specific language governing
+ permissions and limitations under the License. Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file. Names of the original
+ copyright holders (individuals or organizations) should be listed with a '*' in the first column.
+ People who have contributed from an organization can be listed under the organization that actually
+ holds the copyright for their contributions (see the Gates Foundation organization for an example).
+ Those individuals should have their names indented and be marked with a '-'. Email address can be
+ added optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow <kevin.leyow@modusbox.com>
+
+ --------------
+ ******/
+import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi'
+import { Enum } from '@mojaloop/central-services-shared'
+import { ReformatFSPIOPError } from '@mojaloop/central-services-error-handling'
+import Logger from '@mojaloop/central-services-logger'
+import { AuditEventAction } from '@mojaloop/event-sdk'
+import { forwardConsentsRequest } from '~/domain/consents'
+
+import { getSpanTags } from '~/shared/util'
+import * as types from '~/interface/types'
+
+/**
+ * summary: CreateConsent
+ * description: The HTTP request POST /consents is used to establish consent
+ * between a PISP, DFSP and user.
+ * parameters: body, accept, content-length, content-type, date, x-forwarded-for, fspiop-source,
+ * fspiop-destination, fspiop-encryption,fspiop-signature, fspiop-uri, fspiop-http-method
+ * produces: application/json
+ * responses: 202, 400, 401, 403, 404, 405, 406, 501, 503
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function post(_context: any, request: Request, h: ResponseToolkit): Promise<ResponseObject> {
```

Comment:
@kleyow I think we also need a test for this route in the `unit/index.test.ts`

Reply:
hmm `unit/index.test.ts` seems somewhat redundant...but more tests never hurt.
done.

Reply:
I think it's mainly to test that the routes are being resolved correctly